core/src/main/java/org/jdbi/v3/core/result/ResultBearing.java

```diff
@@ -27,17 +38,6 @@
 import org.jdbi.v3.core.statement.StatementContext;
 import org.jdbi.v3.meta.Beta;
 
-import java.lang.annotation.Annotation;
```

Comment:
We really need to ensure everyone is using a common import order, but until we get there, let's try to avoid unnecessary shuffling.  It detracts from the PR and causes code churn.

Reply:
I've been thinking recently I'd like to give an auto-formatter plugin a try.

Reply:
Consider this my vote for Google java style guide